src/pages/OrganizerAddModal.tsx

```diff
@@ -207,13 +205,38 @@ const OrganizerAddModal = ({
           id="organizer-url"
           label={t('organizer.add_modal.labels.url')}
           info={
-            <Alert variant={AlertVariants.PRIMARY}>
-              {t('organizer.add_modal.url_requirements')}
-            </Alert>
+            formState.errors.url?.type === 'not_unique' ? (
+              <Alert variant={AlertVariants.WARNING}>
+                <Trans
+                  i18nKey={`organizer.add_modal.validation_messages.url_not_unique`}
+                  values={{
+                    organizerName: existingOrganization?.name[i18n.language],
+                  }}
+                  components={{
+                    setOrganizerLink: (
+                      <Button
+                        variant={ButtonVariants.UNSTYLED}
+                        onClick={() => onSetOrganizer(existingOrganization)}
+                        css={`
+                          display: inline-block;
+                          font-weight: bold;
+                          text-decoration: underline;
+                          padding: 0;
```

Comment:
I think most of these css props can be added via props.
```suggestion
                      fontWeight="bold"
                      ...
```